src/asm/warning.cpp

```diff
@@ -343,6 +344,9 @@ void error(char const *fmt, ...)
 	va_start(args, fmt);
 	printDiag(fmt, args, "error: ", ":\n    ", NULL);
 	va_end(args);
+
+	if (nbErrors + 1 == maxErrors)
+		errx("Reached %u error%s; assembly aborted!", maxErrors, maxErrors == 1 ? "" : "s");
 	nbErrors++;
```

Comment:
How about:
```suggestion
    // This intentionally makes 0 never enter the condition, so that it acts as "unlimited".
	nbErrors++;
	if (nbErrors == maxErrors)
		errx("The maximum of %u error%s was reached, aborting!", maxErrors, maxErrors == 1 ? "" : "s");
```